src/components/_templates/Orders/OrderList/OrderList.tsx

```diff
@@ -0,0 +1,70 @@
+import { List, Select } from 'antd'
+import React, { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+import apiClient from 'src/helper/api'
+import { useAppDispatch } from 'src/redux/hooks'
+import { loadOrdersAsync } from 'src/redux/orders/orderThunk'
+import { IOrder, IOrdersState } from 'src/redux/orders/type'
+import { CONSTANTS_TEXT, OPTIONS } from 'src/Text'
+import { Title } from 'src/Text/Text'
+import { AddOrders } from '../AddOrders'
+import './OrderList.scss'
+
+export const OrderList = ({ orders }: IOrdersState) => {
+  const dispatch = useAppDispatch()
+
+  const [filteredOrders, setFilteredOrders] = useState<IOrder[]>()
+
+  useEffect(() => {
+    dispatch(loadOrdersAsync())
+  }, [])
+
+  useEffect(() => {
+    apiClient()
+      .get('orders?status=in+storage')
+      .then((response) => {
+        setFilteredOrders(response.data)
+      })
+  }, [orders])
+
+  const handleChange = (value: string[]) => {
```

Comment:
its due to comment above. use status in deps. BTW api will return all orders with this status, but not current user orders